_data/projects.yml

```diff
@@ -281,7 +263,15 @@ publicly release, please send an email to <a href=\"mailto:lind-dev@googlegroups
       image: "img/projects/crashsimulator_diagram.png"
       site:
       status: *stealth
-      description: "CrashSimulator is a tool that replicates “real-world” testing for new and upgraded software without the complications of “real-world” deployment. The program enables software developers to identify vulnerabilities in product designs long before they are packaged and released."
+      description: "CrashSimulator is a tool that replicates “real-world” testing for new and upgraded software without 
+      the complications of “real-world” deployment. The tool is built on an easy-to-use method for detecting potential
```

Comment:
I would not call it an easy-to-use method.  It generates tests without the developer doing per-project work.  When you say easy-to-use, I'm not sure which part of the process (anomaly capture, anomaly replay, developer using the tool, etc.) this refers to.

Reply:
Fixed